config/locales/en/admin.yml

```diff
@@ -31,6 +31,7 @@ en:
       search:
         error: Please enter a search criteria
     settings:
+      notice: "Settings have been updated"
```

Comment:
Do we need a consensus on use of quotes in locales or not. default for yaml is that if you do not use any quotes it is the same as if you had used single quotes. double quotes are only needed if you the translation needs interpolation.

my personal preference is do not use quotes at all in the yaml unless you need to for interpolation or escaping